public/ng/less/gogs/repository.less

```diff
@@ -795,3 +795,84 @@
         padding-top: 15px;
     }
 }
+
+#stars, #watchers, #forks {
```

Comment:
Does it really make sense to add that much custom CSS for this simple kind of pages?